src/styles/dictionary_modal.scss

```diff
@@ -71,6 +71,7 @@
 #desc {
     text-align: left;
     font-weight: 400;
+    word-wrap: break-word;
```

Comment:
looked at the screen shots on the Jira and found it fine

Reply:
Did you manage to look at ones in this ticket?
https://issues.openmrs.org/browse/OCLOMRS-674